lib/web_ui/lib/src/engine/semantics/scrollable.dart

```diff
@@ -28,11 +28,31 @@ import 'semantics.dart';
 /// viewport "scrollTop" may take positive values.
 class Scrollable extends RoleManager {
   Scrollable(SemanticsObject semanticsObject)
-      : super(Role.scrollable, semanticsObject);
+      : super(Role.scrollable, semanticsObject) {
+        _scrollOverflowElement.style
+        ..position = 'absolute'
+        ..transformOrigin = '0 0 0'
+        // Ignore pointer events since this is a dummy element.
+        ..pointerEvents = 'none';
+        semanticsObject.element.append(_scrollOverflowElement);
+      }
 
   /// Disables browser-driven scrolling in the presence of pointer events.
   GestureModeCallback? _gestureModeListener;
 
+  /// Dummy element used as a workaround for issue
+  ///
+  /// When the assistive technology gets to the last element of the scrollable
+  /// list, the browser thinks we don't have any more content, so overrides
```

Comment:
nit: s/so overrides/so it overrides/
nit: our isn't consistent about this, but we avoid "we" in doc comments, as it's not clear what it refers to. In this case, "we" can be replaced with "the scrollable area".